contracts/core/Merge.sol

```diff
@@ -13,17 +13,22 @@ contract Merge is IMerge, IWeweReceiver, IERC1363Spender, Ownable, ReentrancyGua
     using SafeERC20 for IERC20;
 
     IERC20 public immutable wewe;
-    IERC20 public immutable vult;
+    IERC20 public immutable vault;
```

Comment:
this change does not apply, it is not a vault, but refers to the vultisig vult token https://vultisig.com/.

Reply:
Ok, ill spell it out so its clear.